modules/core/src/lib/effect.ts

```diff
@@ -11,7 +11,9 @@ export type PostRenderOptions = LayersPassRenderOptions & {
 export interface Effect {
   id: string;
   props: any;
+  useInCollide?: boolean;
```

Comment:
I would prefer the core to not have any knowledge of this extension. If this needs to be hardcoded anyways, hard code it in the CollideEffect, not in the core.